inc/planning.class.php

```diff
@@ -1247,8 +1247,9 @@ static function showSingleLinePlanningFilter($filter_key, $filter_data, $options
          $group->getFromDB($actor[1]);
          $title = $group->getName();
       } else if ($filter_data['type'] == 'event_filter') {
-         if (!$filter_key::canView()
-               || !($item = getItemForItemtype($filter_key))) {
+         if (!($item = getItemForItemtype($filter_key))) {
+            return false;
```

Comment:
if the ``!canView`` need to break first, just invert the two statements, no need for additional verbosity

Reply:
I have tested.No, same problem

Reply:
ok fine